### PR TITLE
[chore] Add save button in edit modal for filters

### PR DESCRIPTION
**Pull request by theBeginner86**
**Notes for Reviewers**

Screenshot 2023-06-30 at 9 04 32 PM

This PR fixes #

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by theBeginner86** [comment]
// @leecalcote @Abhishek-kumar09

File: ui/components/MesheryFilters/FiltersGrid.js [hunk 1]
```diff
@@ -31,7 +31,8 @@ function FilterCardGridItem({ filter, handleDeploy, handleUndeploy, handleSubmit
         handleDeploy={handleDeploy}
         handleUndeploy={handleUndeploy}
         handleClone={handleClone}
-        deleteHandler={() => handleSubmit({ data : yaml, id : filter.id, type : FILE_OPS.DELETE ,name : filter.name })}
+        deleteHandler={() => handleSubmit({ data : yaml, id : filter.id, type : FILE_OPS.DELETE, name : filter.name })}
+        updateHandler={() => handleSubmit({ data : yaml, id : filter.id, type : FILE_OPS.UPDATE ,name : filter.name })}
```
Comment (leecalcote) [inline ui/components/MesheryFilters/FiltersGrid.js:35]: Has the data: yaml been remapped to the new Component / Config field?

File: ui/components/MesheryFilters/FiltersGrid.js [hunk 2]
```diff
@@ -14,7 +14,8 @@ const INITIAL_GRID_SIZE = { xl : 4, md : 6, xs : 12 };
 
 function FilterCardGridItem({ filter, handleDeploy, handleUndeploy, handleSubmit, setSelectedFilters, handleClone, handlePublishModal, handleUnpublishModal, canPublishFilter }) {
   const [gridProps, setGridProps] = useState(INITIAL_GRID_SIZE);
-  const [yaml, setYaml] = useState(filter.filter_file);
+  console.log("filter: ", filter);
```
Comment (theBeginner86) [inline ui/components/MesheryFilters/FiltersGrid.js:17]:
```suggestion

```

File: ui/components/MesheryFilters/FiltersGrid.js (hunk 1 above)
Comment (theBeginner86) [reply ui/components/MesheryFilters/FiltersGrid.js]: Yes, I've mapped it now.

Comment (theBeginner86) [reply ui/components/MesheryFilters/FiltersGrid.js]: Here: https://github.com/meshery/meshery/pull/7992/files#diff-6ec8e150ffa5db5b0830cd1598d529bbc98c79c98df32092858a51a779fcdea0R693

**Pull request merged**